app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class variable `people` to store instances of `Person` by name can lead to issues if names are not unique. Consider using a unique identifier for each person, such as an ID, to avoid potential conflicts.